docs/fundamentals/executor/executor-api.md

```diff
@@ -294,6 +294,30 @@ Text: hello world
 ```
 
 
+You can even pull an `Executor` from the jina hub and use it directly as a python object.
+
+
+````{admonition} More about the hub? 
+:class: information
+The [hub](https://hub.jina.ai/) is a marketplace for `Executor`.
+````
```

Comment:
```suggestion
````{admonition} More about Jina Hub 
:class: information
The [hub](https://hub.jina.ai/) is our marketplace for `Executor`s.
````
```